assets/src/components/drawer.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { PropsWithChildren, useState } from "react"
+import DrawerTab from "../components/drawerTab"
+import { joinClasses } from "../helpers/dom"
+
+interface Props {
+  open?: boolean
+  onToggleOpen?: () => void
+}
+
+const Drawer = ({
+  open = true,
+  onToggleOpen,
+  children,
+}: PropsWithChildren<Props>) => (
+  <>
+    <div
+      className={joinClasses([
+        "c-drawer",
+        ...(open
+          ? ["c-drawer--visible"]
+          : ["c-drawer--hidden", "u-hideable--hidden"]),
+      ])}
+    >
+      <DrawerTab
+        isVisible={open}
+        toggleVisibility={onToggleOpen || (() => {})}
+      />
+      {children}
+    </div>
+    <button onClick={onToggleOpen} className="c-drawer__backdrop-button" />
+  </>
+)
+
+interface DrawerWithStateProps {
+  startOpen?: boolean
+}
+
+export const DrawerWithState = (
```

Comment:
I don't think this should be exported, and `Drawer` should be exported?

Reply:
I saw your changes to the later PR first, so I fixed it [there](https://github.com/mbta/skate/pull/2844/commits/b8b1d4b873bc11666f13b650c780089eb2f8b69d)